numpy/core/tests/test_multiarray.py

```diff
@@ -4080,7 +4081,10 @@ def test___array__(self):
 class TestResize(TestCase):
     def test_basic(self):
         x = np.array([[1, 0, 0], [0, 1, 0], [0, 0, 1]])
-        x.resize((5, 5))
+        if IS_PYPY:
```

Comment:
Out of curiousity, I noticed the import also defines `HAS_REFCOUNT`, is this reflected in the C-api so that this could be not PyPy specific but in principle good for all non-refcounted pythons?